waspc/data/Generator/templates/react-app/src/test/vitest/helpers.tsx

```diff
@@ -26,10 +26,11 @@ export function renderInContext(ui: ReactElement): RenderResult {
 }
 
 type QueryRoute = Query<any, any>['route']
+type MockQuery = <Input, Output, MockData extends Output>(query: Query<Input, Output>, resJson: MockData) => void
```

Comment:
@shayneczyzewski I've changed this signature to improve type safety. Check [this playground](https://www.typescriptlang.org/play?#code/C4TwDgpgBAigrhATiAPASQHZjsANFAeR22AD4oBeKAbwCgoGoAKAQ0QHMBnALikxICUvAAqIA9gFsAlpwgoiwEqVoBfWrQAmEAMYAbNtABmcDNuBSxGKBLHaA1vCSp+OfApL4AsrbtQIAD2AIDA1OQmIcUiYARwRkXkdkdCxXcMVI-EQITgApTkteb3shKAA3MSkNAG51ALAxRGAoUEgoABUWTl8qOgZK3gw4CQAjJHooLU5tRCkwcwKoTmAZjHZxmQARSwheYbExXQgWDHG4WUQ0DQGh0cRVdS09AyhtSyWodghgDq6eWDjUOVKvgfnYANoAXWUj30WRebyawE6dj+oMhmh0sOgrww7zAbHMLF0oL+wik9hQoPwACJKtSoAAfKDUybTWbzDD0pm0zhbDAQamkdG0Gz2RIgJifb7Izj4JG-ASMJUMAD0KuaAAsZIsNWI4LoNM1wNiNTo7PhSkTKtYfCKfOLJV8Sfh8Y0pESSYqoGrNdrOLr9YaMGJEcaXqb7PgpBhLbpraK7A9Mc8ce8pQR+W0AO5iAiINoarI7f5OFBLFbsfAARkZUAATLWAMzQ5Nw1NNbZ57NiXg1pl1jFPNsIqDbbsFot5gBiesQvdrDaZjdrABZ1AmHenMzmu4WIBB8J38zmvcrGD7gFqwv69Qaja1tBHzWUrYaE3axQDHcAMxBu7ui0PbcxAnfdp1nRULyvHVbyDEN7xNM0owwFhSjjN8fCAA) for the demo.

Reply:
Sounds good to me, thanks!